packages/orchestrator-ui-components/src/components/WfoStartButton/WfoStartWorkflowComboBox.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+import { useRouter } from 'next/router';
+
+import { useCheckEngineStatus } from '@/hooks';
+import { useGetWorkflowOptionsQuery } from '@/rtk';
+import { StartComboBoxOption } from '@/types';
+
+import { PATH_START_NEW_WORKFLOW } from '../WfoPageTemplate';
+import { WfoStartButtonComboBox } from './WfoStartButtonComboBox';
+
+export const WfoStartWorkflowButtonComboBox = () => {
```

Comment:
This component is almost identical to the `WfoStartTaskComboBox.tsx` component. I would suggest to create a generic one.